### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by pombreda** [clhunsen/cppstats#14]
Hi: what would be the license for this code?
Thanks!

**Comment by clhunsen** [comment]
The license is to be LGPL v3. We still have to alter the repository accordingly, however.

**Comment by xai** [comment]
OK, so what do we need to make it happen. An agreement of all previous contributors? I can try to identify them if it helps.

**Comment by pombredanne** [comment]
FWIW, https://github.com/clhunsen/cppstats/blob/master/lib/enum.py reports itself as GPL which may override the LGPL here? 

and other than that only pstat.py and stats.py seem to have a license

As far as contributors I can see only @clhunsen and @joliebig

**Comment by clhunsen** [comment]
Sorry for the delay, but due to time constraints, I wasn't able to add the license earlier. Any license-related changes I have made recently are discussed and agreed on with @joliebig.

Just some minutes ago, I pushed a new version (until now, only on branch `dev`) that removed the Python dependencies that have been shipped before and would have violated the license LGPLv3 (`statlib`, `enum`, and `pyparsing` are now installed via `setup.py`). Same for all `srcML` binaries.

After that, I added the LGPLv3 files and file headers. Any thought and/or confirmation on this is appreciated.
